homeassistant/components/climate/ephember.py

```diff
@@ -96,9 +105,16 @@ def target_temperature_step(self):
     @property
     def current_operation(self):
         """Return current operation ie. heat, cool, idle."""
-        if self._zone['isCurrentlyActive']:
-            return STATE_HEAT
-        return STATE_IDLE
+        return STATE_AUTO
+
+    @property
+    def operation_list(self):
+        """List of available operation modes."""
+        return [STATE_AUTO]
+
+    def set_operation_mode(self, operation_mode):
```

Comment:
What use is this method if it never does anything? How does this make the ephember device support "operation mode setting"?

Reply:
The `set_operation_mode` in the base class throws an exception if it is not implemented here. As it only supports one mode this set operation becomes a do nothing operation to prevent the throw